src/useSubscription/models/channel.ts

```diff
@@ -163,6 +165,7 @@ export default class Channel<T extends Action = Action> {
 
     if (this.subscriptions.size == 0) {
       this.manager.deleteChannel(this.signature)
+      this.scope.stop()
```

Comment:
if children subscriptions aren't properly disposed with parent, it could break this logic to only cleanup channel with all subscriptions are closed

Reply:
I'm not following this comment. The action is executed by the channel. And the channel remains "open" until the last subscription unsubscribes. So when the channel is being deleted the scope is stopped which will trigger the dispose in any effects (the child subscriptions).